src/app/views/shared/services/library-menu.service.ts

```diff
@@ -126,6 +126,9 @@ export class LibraryMenuService {
     }, {
       name: 'Buttons',
       link: LibraryItemSegmentPaths.buttons
+    }, {
+      name: 'Datepicker',
```

Comment:
I think Datepicker should still be hidden until the Angular component is ready...
...and the value of `LibraryItemSegmentPaths.datepicker` should be Datepicker, not DatePicker